server_spidercast/src/MCC_RUM/src/llmLogUtil.c

```diff
@@ -1064,9 +1064,10 @@ static size_t formatMessage(char * msgBuff, size_t msgBuffSize, const char * msg
                     break;
                 default:                 /* Just put the comma into the extension */
                     part++;
-                    if (xleft > 0)
+                    if (xleft > 0){
                         *xp++ = ch;
                         xleft--;
+                    }
```

Comment:
I'm slightly worried about this

Reply:
Made this change to resolve the misleading-indentation warning. I agree it might need more thought, I'm open to discuss better approach for this code change

Reply:
It looks right to me